src/Microsoft.SqlTools.ServiceLayer/SqlProjects/Contracts/DatabaseReferences/AddDacpacReferenceParams.cs

```diff
@@ -0,0 +1,32 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.ServiceLayer.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.SqlProjects.Contracts
+{
+    /// <summary>
+    /// Parameters for adding a Dacpac reference to a SQL project
+    /// </summary>
+    public class AddDacpacReferenceParams : AddDatabaseReferenceParams
+    {
+        /// <summary>
+        /// Path to the .dacpac file
+        /// </summary>
+        public string DacpacPath { get; set; }
```

Comment:
do we enforce this being a relative path to the .sqlproj? That's what we did when adding support for dacpac references in ADS to avoid having full paths to dacpacs in .sqlprojs get checked into source control

Reply:
Not in the DacFx library 'cause absolute paths are technically still valid, and that's meant to be general-purpose.  Think we should convert to relative paths at either the STS or ADS level?

Reply:
ADS makes sense since the paths are coming from there anyway, just have to remember to do that later :)